src/warnet/cli/graph.py

```diff
@@ -128,6 +129,12 @@ def validate(graph: Path):
     """
     Validate a <graph file> against the schema.
     """
-    with open(graph) as f:
-        graph = nx.parse_graphml(f.read(), node_type=int, force_multigraph=True)
+    try:
+        with open(graph) as f:
+            graph = nx.parse_graphml(f.read(), node_type=int, force_multigraph=True)
+    except ParseError as e:
+        print(f"Error. {e}")
+        print("The graph file appears malformed. Pleases consider generating a graph file with:")
+        print("  warcli graph create")
```

Comment:
10ecff1afbb2ce2efe91044cd5c4c872219c68f1

again i think hard coding suggested commands in error messages could lead to issues

Reply:
Removed here: https://github.com/mplsgrant/warnet/blob/a68a229af6e90e4111d1ef6461cc8670cc1cdce8/src/warnet/cli/graph.py#L146